snap/snapcraft.yaml

```diff
@@ -13,13 +13,17 @@ confinement: strict
 
 parts:
     novnc:
-        source: build/
+        source: https://github.com/novnc/noVNC/archive/v1.2.0.tar.gz
         plugin: dump
         stage-packages:
-            - websockify
             - bash
             - jq
-            - python-numpy
+            - python3-numpy
```

Comment:
Why do we need this change all of a sudden?

Reply:
python-numpy is the Python 2 package (Ubuntu apt package). I realised it's not necessary, only python3-numpy is.

Reply:
Actually python3-numpy can probably go too, if it's not something novnc itself uses. I had those python*-numpy packages in there as they seemed to be dependencies of the websockify apt package, which is now no longer used

Reply:
Indeed. We have no python in noVNC, so it's strictly a websockify thing.

Reply:
In that case, python3-numpy can be removed too